rekognition_tutorial/short_sa_rekognition_tutorial.ipynb

```diff
@@ -0,0 +1,1222 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1468d369",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def show_custom_labels(client, model, bucket, photo, min_confidence):\n",
+    "    response = client.detect_custom_labels(Image = {'S3Object': {'Bucket': bucket, \n",
+    "                                                                 'Name': photo}},\n",
+    "                                           MinConfidence = min_confidence,\n",
+    "                                           ProjectVersionArn = model)\n",
+    "    return response['CustomLabels']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f5be1fd6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = 'FILE TO TEST'\n",
+    "photo = f'{BUCKET_FOLDER_PATH}/unlabeled/{filename}'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "68398604",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "min_confidence = 10 ## minimum confindence model should have for the label to detect it\n",
+    "\n",
+    "%time labels = show_custom_labels(rek_client, model_arn, bucket_name, photo, min_confidence)\n",
+    "labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b6d2788",
+   "metadata": {},
+   "source": [
+    "## Predict unlabeled images\n",
+    "<a id='predict_unlabeled_images'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5134cf3e",
+   "metadata": {},
+   "source": [
+    "We can now get the predictions for all our unlabeled images.\n",
+    "\n",
+    "We could use the function $\\textit{show_custom_labels}$ and dictionary with S3 paths to all unlabeld images $data\\_ links\\_dict$ that we created above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a9314fd6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predicted_labels  = {}\n",
+    "\n",
+    "for photo_path in tqdm(data_links_dict['unlabeled']):\n",
+    "    labels = show_custom_labels(rek_client, model_arn, bucket, photo_path, min_confidence)\n",
+    "    predicted_labels[photo_path] = labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5fc94a3",
+   "metadata": {},
+   "source": [
+    "## Make SA annotations\n",
+    "<a id='make_sa_annotations'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14f80c2f",
+   "metadata": {},
+   "source": [
+    "Based on predictions made by the model we should now create annotations in SuperAnnotate format to be able to upload them to SuperAnnotate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f92fb0d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ANNOTATIONS_FOLDER = 'PATH/TO/LOCAL/DIR/' # local folder to store .json files with annotations\n",
+    "for image_path, label in predicted_labels.items():\n",
+    "    filename = image_path.split('/')[-1]\n",
```

Comment:
Let's use os module, here and everywhere.

os.path.basename

Reply:
+